pkgs/top-level/all-packages.nix

```
@@ -26829,7 +26829,9 @@ with pkgs;
 
   jwm-settings-manager = callPackage ../applications/window-managers/jwm/jwm-settings-manager.nix { };
 
-  k3s = callPackage ../applications/networking/cluster/k3s {};
+  k3s = callPackage ../applications/networking/cluster/k3s {
+    buildGoModule = buildGo117Module;
+  };
```

```suggestion
  k3s = callPackage ../applications/networking/cluster/k3s { };
```
buildGo117Module can be references directly at `k3s/default.nix`. No need to change top level.

---

Top level is usually preferred if it only needs to be set once and doesn't need any other changes for it to work.

---

Since https://github.com/NixOS/nixpkgs/pull/154059 merged, the default is now `buildGo117Module`, so I've just removed the override entirely.